websites/S/sekolahmu/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.13",
+  "apiVersion": 1,
+  "author": {
+    "id": "999863217557880842",
+    "name": "vintheweirdass"
+  },
+  "service": "sekolahmu",
+  "altnames": ["sekolah.mu", "smm"],
+  "description": {
+    "en": "Sekolah.mu is a education platform for all ages with the blended learning approach",
+    "id": "Sekolah.mu adalah platform edukasi untuk semua kalangan dengan pendekatan blended learning"
```

Maybe use the site's description. (taken from page source)
`Belajar dari mana saja dengan sistem blended learning, menghadirkan program pilihan untuk murid, sesuai dengan fokus dan impian yang ingin diraih.`

or if you want to keep this description, update it to this:
```suggestion
    "en": "Sekolah.mu is an education platform for all ages using the blended learning approach",
    "id": "Sekolah.mu adalah platform edukasi untuk semua kalangan dengan pendekatan blended learning"
```

---

hmm, ill accept this

---

since the site description is kinda like an 'ad'